modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -831,6 +832,30 @@ describe('V2 Wallet:', function () {
         preBuiltSignedTx.should.have.property('txHex');
       });
     });
+
+    it('Should build token enablement transactions correctly', async function () {
+      const params = {
+        tokens: [{ name: 'tsol:usdc' }, { name: 'tsol:srm' }, { name: 'tsol:gmt' }],
+      };
+      nock(bgUrl)
+        .post(`/api/v2/wallet/${solWallet.id()}/txrequests`)
+        .reply((url, body) => {
+          const bodyParams = JSON.parse(body);
+          bodyParams.intent.intentType.should.equal('enableToken');
+          bodyParams.intent.recipients.length.should.equal(0);
+          bodyParams.intent.tokens.should.deepEqual(params.tokens);
+          return {
+            apiVersion: 'full',
+            transactions: [{
+              unsignedTx: {
+                serializedTxHex: 'fake transaction',
+                feeInfo: 'fake fee info',
+              },
+            }],
+          };
+        });
+      await solWallet.buildTokenEnablements(params);
```

Comment:
let's verify the nock so we know your `reply` body was actually executed.

should be able to do something like:
```
const txRequestNock = nock(bgUrl)...

await solWallet.buildTokenEnablements(params);

txRequestNock.isDone();
```